routes/students.js

```diff
@@ -2,6 +2,7 @@
 
 const Promise = require('bluebird');
 const query = require('../lib/utils').query;
+const defined = require('../lib/utils').defined;
```

Comment:
Instead of requiring utils twice, require it once into a variable called utils and pull the two functions you want off of it. This is cleaner and more efficient.